protos/feast/core/CoreService.proto

```diff
@@ -90,27 +90,29 @@ service CoreService {
 
     // Lists all projects active projects.
     rpc ListProjects (ListProjectsRequest) returns (ListProjectsResponse);
-  
+
+    // Internal API for Job Coordinator to update featureSet's status once responsible ingestion job is running
+    rpc UpdateFeatureSetStatus(UpdateFeatureSetStatusRequest) returns (UpdateFeatureSetStatusResponse);
```

Comment:
By the way, we can add a space between the name of the RPC and the opening bracket so that colouring on GitHub works.